.gitignore

```diff
@@ -2,3 +2,4 @@ node_modules
 dist/bundle.js
 dist/*.js.*
 dist/*.png
+.idea
```

Comment:
why is this needed?

Reply:
Lol, that's added by my IDE ... so I need to ignore it

Reply:
it's ok, why IDE you're using?

Reply:
I'm using Webstorm :)

Reply:
💰 💰 💰 💰 💰